packages/mediacenter/kodi/package.mk

```diff
@@ -229,14 +229,13 @@ export CXXFLAGS_FOR_BUILD="$HOST_CXXFLAGS"
 export CFLAGS_FOR_BUILD="$HOST_CFLAGS"
 export LDFLAGS_FOR_BUILD="$HOST_LDFLAGS"
 
-export PYTHON_VERSION="2.7"
```

Comment:
GitHub doesn't let me comment on random lines, but lines 374-375 still reference `$PYTHON_VERSION`.
`kodi-send` is currently broken in Milhouse builds ("No module named xbmcclient") as its imports get installed in
`/usr/lib/python/site-packages/kodi` (vs `python2.7`)